sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Models/QueryAggregateSeriesRequestOptions.cs

```diff
@@ -14,7 +14,7 @@ public class QueryAggregateSeriesRequestOptions : QueryRequestOptions
         /// Selected variables that needs to be projected in the query result. When it is null or not set, all the
         /// variables and Time Series types in the model are returned.
         /// </summary>
-        public List<string> ProjectedVariables { get; }
+        public IList<string> ProjectedVariableNames { get; }
```

Comment:
If we provide an interface here why not other places such as ProjectedProperties in sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/Models/QueryEventsRequestOptions.cs?